run-build-functions.sh

```diff
@@ -72,6 +72,18 @@ mkdir -p $NETLIFY_CACHE_DIR/.cargo
 : ${NPM_FLAGS=""}
 : ${BUNDLER_FLAGS=""}
 
+# Feature flags are a comma-separated list.
+# The following logic relies on the fact that feature flags cannot currently
+# have escaped commas in their value. Otherwise, parsing the list as an array,
+# e.g. using `IFS="," read -ra <<<"$1"` would be needed.
+has_feature_flag() {
```

Comment:
Not used yet, but I'll follow up with a PR to replace `EXPERIMENTAL_NPM_WORKSPACES_CACHING` by it.